UI/src/view/navigationpanel/pointAssetBox.js

```diff
@@ -0,0 +1,115 @@
+(function (root) {
+  root.PointAssetBox = function (assetConfig) {
+    ActionPanelBox.call(this);
+    var me = this;
+
+    this.header = function(){
+      return assetConfig.title;
+    };
+
+    this.title = function () {
+      return assetConfig.title;
+    };
+
+    this.layerName = function () {
+      return assetConfig.layerName;
+    };
+
+    this.elements = function (){
+      return { expanded: $([
```

Comment:
This should be checked for point assets! The div closes in the different place as all the other assets.

Reply:
This div is used to separate the checkbox from the labeling